packages/react-native-executorch/common/rnexecutorch/models/speech_to_text/SpeechToText.cpp

```diff
@@ -94,12 +97,18 @@ void SpeechToText::stream(std::shared_ptr<jsi::Function> callback,
     }
     ProcessResult res =
         this->processor->processIter(DecodingOptions(languageOption));
-    nativeCallback(res.committed, res.nonCommitted, false);
+
+    std::vector<char> committedVec(res.committed.begin(), res.committed.end());
+    std::vector<char> nonCommittedVec(res.nonCommitted.begin(), res.nonCommitted.end());
+
+    nativeCallback(committedVec, nonCommittedVec, false);
     this->readyToProcess = false;
   }
 
   std::string committed = this->processor->finish();
```

Comment:
Maybe we change signature (so it will return `std::vector<char>` instead of `std::string`) of `this->processor->finish` and what is returned in `ProcessResult` if these functions are used only here. If used anywhere else as `std::string` we should probably keep it as it is